app/controllers/users_controller.rb

```diff
@@ -0,0 +1,75 @@
+# frozen_string_literal: true
+
+# Handle user CRUD.
+class UsersController < ApplicationController
+  before_action :set_user, only: %i[show edit update destroy]
+
+  # GET /users.
+  # List Users.
+  def index
+    @users = User.all
+  end
+
```

Comment:
Remove extra blank line

Reply:
When edited with vi there is no extra line.